.github/workflows/build-wheels.yml

```
@@ -0,0 +1,44 @@
+name: Builds
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+  # Run weekly at 1:23 UTC
+  schedule:
+  - cron: '23 1 * * 0'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository and submodules
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+
+      - name: Display Python version
+        run: python --version
+
+      - name: Build SDist and wheel
+        run: pipx run build
+
+      - name: Upload to GitHub
+        uses: actions/upload-artifact@v4
+        with:
+          name: Packages
+          path: dist/*
+
+      - name: Check metadata
+        run: pipx run twine check dist/*
```

```suggestion
        run: pipx run twine check --strict dist/*
```

`twine` has a strict option.

```console
$ pipx run twine check --help
usage: twine check [-h] [--strict] dist [dist ...]

positional arguments:
  dist        The distribution files to check, usually dist/*

options:
  -h, --help  show this help message and exit
  --strict    Fail on warnings
```